server/impl/src/main/java/com/walmartlabs/concord/server/process/queue/FilterUtils.java

```diff
@@ -55,7 +55,9 @@ public final class FilterUtils {
             SuffixMapping.of(".notEndsWith", ProcessFilter.FilterType.NOT_ENDS_WITH),
 
             SuffixMapping.of(".ge", ProcessFilter.FilterType.GREATER_OR_EQUALS),
-            SuffixMapping.of(".len", ProcessFilter.FilterType.LESS_OR_EQUALS_OR_NULL)
+            SuffixMapping.of(".len", ProcessFilter.FilterType.LESS_OR_EQUALS_OR_NULL),
+
+            SuffixMapping.of(".rm", ProcessFilter.FilterType.REGEXP_MATCH)
```

Comment:
`.regex` maybe? A bit more self-explanatory. Won't be the longest suffix there too. :-)